users/src/api/routes/users.routes.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
+const { filter } = require('../../util');
+
+const router = express.Router();
+const UserService = require('./../services/user.service');
+const service = new UserService();
+
+router.get('/',
+    async (req, res, next) => {
+        try{
+            let users = await service.find();
+            console.log('🟢 User list request');
+            users = filter(
+                ['id','name','username','email','role'],
+                users
+            );
+            return res.status(200).json(users).end();
+        } catch(error) {
+            next(error);
+        } 
+    }
+);
+
+router.get('/:id',
+    async (req, res, next) => {
+    try {
+        const { id } = req.params;
+        const user = await service.findOne(id);
+        res.json(user);
+    } catch (error) {
+        next(error);
+    }
+});
+
+
+router.post("/",
+    async (req, res, next) => {
+        try {
+            const user = req.body;
+            const newUser = await service.create(user);
+            res.status(201).json(newUser);
+        } catch (error) {
+            next(error);
+        }
+});
+
+router.patch('/:id',
+    async (req, res, next) => {
+    try {
+        const { id } = req.params;
+        const body = req.body;
+        let user = await service.findOne(id);
+        
+        user = {
+            name: body.name || user.name,
+            username: body.username || user.username,
+            email: body.email || user.email,
+            role: body.role || user.role
+        };
+
+        await service.update(id, user);
+        res.json(user);
+
+    } catch (error) {
+        next(error);
+    }
+});
+
+router.delete('/:id',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ce-box/CE5508-astraoke-api/pullRequest?prid=8478842)
```suggestion
router.delete("/:id",
```